crates/re_viewport/src/context_menu/actions/remove.rs

```diff
@@ -48,9 +47,9 @@ impl ContextMenuAction for RemoveAction {
         instance_path: &InstancePath,
     ) {
         if let Some(space_view) = ctx.viewport_blueprint.space_view(space_view_id) {
-            space_view.contents.add_entity_exclusion(
```

Comment:
I think it would be wise to also remove the old foot-gunny `add_entity_exclusion`, or rewrite it to never include the subtree (i.e. take an entity path instead of a `EntityPathRule`)

Reply:
I considered it, but I found the behavior that it gives in the query editor-view to still be useful.

In that view we give much clearer visibillity into all the rules that are in effect and the fact that 1 click = 1 rule modification matches what a user also does when hand-editing the expressions, where I would expect them to manually delete the inclusion lines rather than having them deleted automagically once they've typed a subtree removal expression.